fiftyone/core/utils.py

```diff
@@ -1697,36 +1726,51 @@ def default_calculate_size(obj):
 
 def get_default_batcher(
     iterable,
+    batcher=None,
```

Comment:
This doesn't really return a "default" batcher if the user provides a `batcher`, but couldn't really think of an alternative that didn't require renaming this method